### ISSUE TITLE
Please get rid of the svelte warning when using this library

### ISSUE THREAD

**Issue by Khoulaiz** [cristovao-trevisan/svelte-icon#9]
```
10:34:01 AM [vite-plugin-svelte] WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.

svelte-icon@2.0.0

Please see https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition for details.
```
Please update it, should be really easy.

**Comment by cristovao-trevisan** [comment]
There's a need to update all the build tools, since svelte-package has changed the way works.
Meanwhile I've updated manually the .exports in package.json for 2.0.2, please try it out to see if the warning is still there

**Comment by Khoulaiz** [comment]
While version 5 is the current major version of svelte, we are still using svelte 4.
So we upgraded to svelte-icon 2.1.0 and this warning is gone. But in exchange we had to do a force install because of the dependency conflict that is now introduced.
```
❯ npm install --save-dev svelte-icon                                                           ~/p/pwss/frontend/email
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: email@0.0.1
npm ERR! Found: svelte@4.2.15
npm ERR! node_modules/svelte
npm ERR!   dev svelte@"^4.2.15" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer svelte@"^3" from svelte-icon@2.1.0
npm ERR! node_modules/svelte-icon
npm ERR!   dev svelte-icon@"^2.1.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! /Users/vwh2ipn/.npm/_logs/2024-04-19T07_39_34_881Z-eresolve-report.txt
```

svelte-icon works without problems with svelte 4 btw.

**Comment by Khoulaiz** [comment]
Just for information: We use this lib for all kind of SVGs, not just icons. It's the most simplest way to control SVG illustrations in a svelte project. Really great for it. Just to let you know...

**Comment by cristovao-trevisan** [comment]
Had the wrong semver for the peerDepenencies, should be fixed in 2.1.1

**Comment by Khoulaiz** [comment]
I can confirm that 2.1.1 fixed all the warnings. Thanks a lot!
